src/sentry/search/eap/resolver.py

```diff
@@ -692,9 +696,13 @@ def resolve_aggregate(
         alias = match.group("alias") or column
 
         # Get the function definition
-        if function not in self.definitions.functions:
+        function_definition: FunctionDefinition | FormulaDefinition
+        if function in self.definitions.functions:
+            function_definition = self.definitions.functions[function]
```

Comment:
In a future refactor, we can rename `self.definitions.functions` to `self.definitions.aggregates`